source/stories.tsx

```diff
@@ -130,3 +130,53 @@ export const WithMaxHeight: FC = () => {
     </main>
   )
 }
+
+type FunctionRefProps = Record<string, never>
+
+interface FunctionRefState {
+  value: string
+}
+
+class FunctionRef extends Component<FunctionRefProps, FunctionRefState> {
+  el: HTMLTextAreaElement | null = null
+
+  constructor(props: FunctionRefProps) {
+    super(props)
+    this.state = {
+      value: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+    }
+  }
+
+  handleChange = (e: ChangeEvent<HTMLTextAreaElement>) => {
+    this.setState({ value: e.currentTarget.value })
+  }
+
+  setRef = (node: HTMLTextAreaElement): void => {
+    this.el = node
+  }
+
+  render() {
+    console.log(this.el)
```

Comment:
Looks like a stray `log` here?

Reply:
I did this in the example so that on every render you can see in the console that you indeed have a value for `this.el`. It was just _something_ to prove that it was set